java/src/lib.rs

```diff
@@ -33,6 +36,52 @@ mod ffi_test;
 #[cfg(ffi_test)]
 pub use ffi_test::*;
 
+/// Configuration for OpenTelemetry integration in the Java client.
+///
+/// This struct allows you to configure how telemetry data (traces and metrics) is exported to an OpenTelemetry collector.
+/// - `traces`: Optional configuration for exporting trace data. If `None`, trace data will not be exported.
+/// - `metrics`: Optional configuration for exporting metrics data. If `None`, metrics data will not be exported.
+/// - `flush_interval_ms`: Optional interval in milliseconds between consecutive exports of telemetry data. If `None`, a default value will be used.
+///
+/// At least one of traces or metrics must be provided.
+#[derive(Clone)]
```

Comment:
```suggestion
#[derive(Clone, Debug)]
```
This may help you debugging things if needed. Applicable to all structs

Reply:
Updated.